### PR TITLE
added basic styles to location-details-component

### PR DESCRIPTION
**Pull request by marcinel123**
Hi Adam,

I added only basic styles to location detail component but I have a problem with setting the height of this component.

I set the height to 1/5 of the screen and I want to translate it -50%. It sets height correctly and moves the whole div -50% to the top. But there still remains white space between Search and Map.

I attached the screen of this problem.

location_component

Please have a look at this issue.

Thanks,
Martin

### CODE REVIEW

File: src/Components/LocationDetails.tsx [hunk 1]
```diff
@@ -1,3 +1,22 @@
 export const LocationDetails = () => {
-	return <div>LocationDetails</div>;
+	return (
+		<div className="h-1/5 w-3/4 relative -translate-y-1/2 bg-white mx-auto flex justify-evenly items-center rounded-2xl">
+			<div className="flex flex-col">
+				<h3>Ip Address</h3>
```
Comment (AdamKniec) [inline src/Components/LocationDetails.tsx]: Is this really a header?

**Review by AdamKniec — CHANGES_REQUESTED** [review]
We do not merge this PR until the component height issue is resolved.

**Comment by marcinel123** [comment]
Please have a look as I finished location component styling.

regards,
Martin

**Comment by marcinel123** [comment]
Please have a look after all changes.

regards,
Martin

File: src/Components/LocationDetails.tsx (hunk 1 above)
Comment (marcinel123) [reply src/Components/LocationDetails.tsx]: fixed

**Review by AdamKniec — APPROVED** [review]

**Pull request merged**